tests/e2e/pipeline.go

```diff
@@ -412,6 +412,13 @@ func createPipelineRolloutSpec(name, namespace string, pipelineSpec numaflowv1.P
 					Raw: pipelineSpecRaw,
 				},
 			},
+			Strategy: &apiv1.PipelineTypeRolloutStrategy{
+				PipelineTypeProgressiveStrategy: apiv1.PipelineTypeProgressiveStrategy{
+					Progressive: apiv1.ProgressiveStrategy{
+						AssessmentSchedule: "240,60,10",
+					},
+				},
+			},
```

Comment:
By changing it here, it applies it to all tests. I think this makes all tests longer, such as functional-pipeline (progressive).

It will be better if we can just change it for the concurrent test since that's the one which update Numaflow Controller as well.